### PR TITLE
[CPM-125] add button to fill all courses

### PR DESCRIPTION
**Pull request by HongShi10**
## Changes

- Added parser to parse regulations into requirements etc..
- Added component to fill all courses

SPECIAL THANKS TO NISARAG

## Checklist

- [] Have tests been written?
- [x] Is the UI consistent?

## Requests
- Can somebody check the styling/CSS?

Addresses [CPM-125]

[CPM-125]: https://momentum-agile.atlassian.net/browse/CPM-125

### CODE REVIEW

File: api/src/routes/course-route.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ router.get("/", async (req, res) => {
                 res.status(404).json();
             } else {
                 LOGGER.info("GET Request Succeeded for /course");
-                LOGGER.info(courses);
+                // LOGGER.info(courses);
```
Comment (Dinith1) [inline api/src/routes/course-route.js:85]: Is this supposed to be commented?

File: api/src/routes/uni-api-route.js [hunk 2]
```diff
@@ -25,24 +27,52 @@ const uni = require("../uni-service");
...
  *        "400":
  *          description: Error message
  *
  */
 router.get("/programme", async (req, res) => {
     const subject = req.query.subject;
     const year = req.query.year || 2021;
-    const data = await uni.fetchAllCourses(subject, year);
-    if (data.error) {
+    console.log(subject)
```
Comment (Dinith1) [inline api/src/routes/uni-api-route.js:39]: `console.log()` should probably be removed :eyes:

File: api/test/api/uni-route.test.js [hunk 3]
```diff
@@ -28,30 +28,31 @@ beforeEach(() => {
...
-            method: "GET",
-            headers: { "Content-Type": "application/json" },
-        });
-        const respJson = await resp.json();
+        jest.setTimeout(async () => {
+            const resp = await fetch("http://localhost:8081/uni/programme?subject=SOFTENG&year=2021", {
+                method: "GET",
+                headers: { "Content-Type": "application/json" },
+            });
+            console.log(resp)
```
Comment (Dinith1) [inline api/test/api/uni-route.test.js:52]: `console.log()` should probably be removed :eyes:

File: client/src/common/UniClient.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import ApiClient from "./ApiClient";
+
+const createAllUniApiCourses = (subject) => {
+    console.log(subject);
```
Comment (Dinith1) [inline client/src/common/UniClient.js:4]: `console.log()` should probably be removed :eyes:

File: client/src/pages/Courses/AddAllCourse.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    AlertDialogBody,
+    AlertDialogFooter,
+    FormControl,
+    Select,
+} from "@chakra-ui/core";
+
+const AddAllCourse = ({ isOpen, onClose, confirm, navigateTo }) => {
+    const cancelRef = React.useRef();
+    const [subject, setSubject] = useState("");
+    console.log(subject);
```
Comment (Dinith1) [inline client/src/pages/Courses/AddAllCourse.js:17]: `console.log()` should probably be removed :eyes:

**Review by Dinith1 — APPROVED** [review]
Very Noice! 💯 🥇 

:shipit: 

@HongShi10 
@FocalChord 
@Richardng12

**Review by daltonlim — APPROVED** [review]
Lgtm

**Review by FocalChord — APPROVED** [review]
good work

**Pull request merged**